tutorials/tutorial-01.md

```diff
@@ -0,0 +1,117 @@
+# Tutorial 2: Tests for Distributions and samples
+
+For the following exercises and tutorials, you will be building up a codebase in your repository.
+
+During this tutorial, you will copy over some functions from Exercise sheet 1, and modify them to pass the tests that you can find on the bottom of this sheet.
+
+> [!TIP]
+> Reminder on how to run tests in [the setup instructions](https://github.com/RUB-EP1/ExercisesDataAnalysisWS2425/blob/main/exercises/setup.md#back-to-julia-running-tests).
+
+We will need some of the p.d.f.s in un-normalized form (such that the optimizations during fitting can work faster)
+
+1. Gaussian
+2. Polynomial
+3. Relativistic Breit-Wigner
+4. Voigt profile
+
+To pass the tests, their signatures should look like this:
+
+1. `gauss(x; μ, σ, a)`, where `a` is an arbitrary normalization factor
+2. `poly(x; coeffs)`, where `coeffs` is a tuple of coefficients. Hint: implement the polynomials in a loop.
+3. `rbw(x; M, Γ, a)`
+4. `voigt(x; M, Γ, σ, a)`
+
+We will also use the sampling methods and generalize them to take any function as input. Their signature looks like this:
+
+```julia
+function sample_rejection(f, n, support, nbins=1000)
+# f is the function,
+# n the sample size (passing the method),
+# support is a tuple specifying in which range the function will be sampled
+# nbins is the number of equidistant points in "support" for finding the maximum of the function (the default value is 1000)
+```
+
+```julia
+function sample_inversion(f, n, support, nbins=1000)
+# Similar to sample_rejection.
+# Here, nbins is the number of bins in which the c.d.f. is pre-computed. See lecture-02-a for details.
+```
+
+Finally, you will be implementing a method to compute the correlation between two arrays:
+
+```julia
+correlation(A, B)
+```
```

Comment:
@DhruvanshuSan and students found this function already implemented in `Statistics.jl`